### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by 1nsider21**
- [DEMO LINK](https://1nsider21.github.io/react_tabs-with-router/)

### CODE REVIEW

**Comment by irenhh** [comment]
it should work after reloading

File: .eslintrc.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 module.exports = {
-  extends: "@mate-academy/eslint-config-react-typescript",
+  extends: "@mate-academy/eslint-config-react",
   // extends: "@mate-academy/eslint-config-react",
```
Comment (irenhh) [inline .eslintrc.js:3]: ?

**Review by irenhh — COMMENTED** [review]
remove comments

File: .eslintrc.js (hunk 1 above)
Comment (1nsider21) [reply .eslintrc.js:3]: https://prnt.sc/unjmd5
This was in the description of the task

Comment (1nsider21) [reply .eslintrc.js:3]: Screenshot_8
This was in the description of the task

Comment (irenhh) [reply .eslintrc.js:3]: I meant a comment)

**Review by irenhh — APPROVED** [review]

**Pull request closed without merging**